modules/mod_search/mod_search.erl

```diff
@@ -529,11 +529,9 @@ search(_, _, _) ->
 %% @doc Expand a search string like "hello wor" to a PostgreSQL tsquery string.
 %%      If the search string ends in a word character then a wildcard is appended
 %%      to the last search term.
--spec to_tsquery(binary()|string(), #context{}) -> binary().
+-spec to_tsquery(binary(), #context{}) -> binary().
```

Comment:
The spec in line 532 uses a record, please define a type for the record and use that instead.